resteasy-core/src/main/java/org/jboss/resteasy/plugins/providers/ProviderHelper.java

```diff
@@ -160,7 +160,7 @@ public static void writeTo(final InputStream in, final OutputStream out) throws
     */
    public static CompletionStage<Void> writeTo(final InputStream in, final AsyncOutputStream out)
    {
-      final byte[] buf = new byte[2048];
+      final byte[] buf = new byte[AsyncOutputStream.ASYNC_MESSAGE_WRITE_BUFFER_SIZE];

```

Comment:
There are other places in this class where `byte[]` is used in a similar fashion.  If easy, I'd try to verify whether or not similar treatment should be given to them.  _But_ I absolutely wouldn't get bogged down on that.  I'm a 'build what you need' kind of guy!  Maybe @jamezp could help us on this..

Reply:
I'm not too sure what to do with the other buffers. However, the property name seems to be specific for async calls and some of the other buffers aren't explicitly used for async calls.

Reply:
I'm definitely fine with the if-aint-broke-dont-fix-it mentality here!